### PR TITLE
Mars 11 table component

### PR DESCRIPTION
**Pull request by aas89**
(left a space for a loading spinner)

### CODE REVIEW

File: src/components/pages/weather/elements/WeatherColumn.js [hunk 1]
```diff
@@ -6,12 +6,14 @@ import WeatherPressureCell from './WeatherPressureCell';
 
 export default class WeatherColumn extends Component {
   render() {
+    const { earthDate } = this.props.dayWeatherData;
+    const { season } = this.props.dayWeatherData;
```
Comment (jackheywood) [inline src/components/pages/weather/elements/WeatherColumn.js]: Nice object destructuring! You can actually do this in one line:

`const { earthDate, season } = this.props.dayWeatherData;`

or if you're feeling really ambitious you can do it for all of the properties!

`const { earthDate, season, temp, wind, pressure } = this.props.dayWeatherData;`

[Mozilla has some good documentation about this.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)

File: src/components/pages/weather/elements/WeatherHeadingCell.js [hunk 2]
```diff
@@ -2,9 +2,13 @@ import React, { Component } from 'react';
 
 export default class WeatherHeadingCell extends Component {
   render() {
+    const { earthDate } = this.props;
+    const date = new Date(earthDate);
```
Comment (jackheywood) [inline src/components/pages/weather/elements/WeatherHeadingCell.js]: I think object destructuring is a little confusing when it's just one value - much clearer to say `const earthDate = this.props.earthDate;`.

We could also do this in one line using:

`const date = new Date(this.props.earthDate);`

File: src/components/pages/weather/elements/WeatherTable.js [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
+        <WeatherColumn dayWeatherData={this.state.inSightWeather[2]} />
+        <WeatherColumn dayWeatherData={this.state.inSightWeather[3]} />
+        <WeatherColumn dayWeatherData={this.state.inSightWeather[4]} />
+        <WeatherColumn dayWeatherData={this.state.inSightWeather[5]} />
+        <WeatherColumn dayWeatherData={this.state.inSightWeather[6]} />
       </div>
+    ) : (
+      <div>This will be a loading spinner.</div>
+    )
     );
```
Comment (jackheywood) [inline src/components/pages/weather/elements/WeatherTable.js]: I'd probably put both of these brackets on the same line for clarity

**Review by jackheywood — CHANGES_REQUESTED** [review]
Nice work! There's so much stuff here!

I'm a bit conflicted about the `dayWeather...` name - in my mind it implies there should be `nightWeather...` but that could just be me.

Perhaps `dailyWeather...` is a clearer name? Up to you!

File: src/components/pages/weather/elements/WeatherHeadingCell.js (hunk 2 above)
Comment (aas89) [reply src/components/pages/weather/elements/WeatherHeadingCell.js]: I had "const earthDate = this.props.earthDate;" originally but the formatting asks you to restructure it.
I've changed it to the one line version which is much nicer than what I had before.

File: src/components/pages/weather/elements/WeatherColumn.js (hunk 1 above)
Comment (aas89) [reply src/components/pages/weather/elements/WeatherColumn.js]: I've done a vote and I think I'll leave it.

**Review by aas89 — COMMENTED** [review]
I've done a vote and I think I'll leave it.

File: src/components/pages/weather/elements/WeatherHeadingCell.js (hunk 2 above)
Comment (jackheywood) [reply src/components/pages/weather/elements/WeatherHeadingCell.js]: Thanks for letting me know! I've updated the config to not error on that, but I guess it's good if it's taught you about destructuring!

**Review by jackheywood — APPROVED** [review]
Looks great!

**Pull request merged**